orchestrators/ProcurementOperations/src/Services/GrIrAccrualService.php

```
@@ -262,249 +154,64 @@ public function writeOffAccrual(
-            'vendor_id' => $vendorId,
-            'total_open_accruals' => 0,
-            'total_open_amount' => Money::of(0, 'USD'),
-            'buckets' => $bucketResults,
-            'summary' => [
-                'current' => Money::of(0, 'USD'),
-                'overdue' => Money::of(0, 'USD'),
-                'critical' => Money::of(0, 'USD'),
-            ],
-        ];
-    }
+    public function getAccrualsByVendor(string $tenantId, string $vendorId, bool $unmatchedOnly = true): array { return []; }
```

PHPMD (2.15.0)

[error] The method getAccrualsByVendor has a boolean flag argument $unmatchedOnly, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)